trollflow2/plugins/__init__.py

```diff
@@ -421,6 +421,19 @@ def check_platform(job):
             "'%s' not in list of allowed platforms" % platform)
 
 
+def check_sensor(job):
+    """Check if the sensor is valid.  If not, discard the scene."""
+    mda = job['input_mda']
+    product_list = job['product_list']
+    conf = get_config_value(product_list, '/product_list', 'processed_sensors')
+    if conf is None:
+        return
+    sensor = mda['sensor']
+    if sensor not in conf:
+        raise AbortProcessing(
+            "'%s' not in list of allowed sensors" % str(sensor))
```

Comment:
Should we have a generic metadata checker instead ?
so in the product list:
```yaml
...
processed_metadata:
  sensors: ['viirs', 'modis']
  platform: ['EOS-Terra', 'NOAA-20']
```

Reply:
Yes, I'll do that.

Reply:
I refactorized these two to a single generic `check_metadata` plugin. I'll add the relevant documentation to https://github.com/pytroll/trollflow2/pull/75